chromadb/telemetry/events.py

```diff
@@ -13,11 +13,52 @@ class ServerStartEvent(TelemetryEvent):
     name: ClassVar[str] = "server_start"
 
 
+@dataclass
+class ClientCreateCollectionEvent(TelemetryEvent):
+    name: ClassVar[str] = "client_create_collection"
+    collection_uuid: str
+    embedding_function: str
+
+
 @dataclass
 class CollectionAddEvent(TelemetryEvent):
     name: ClassVar[str] = "collection_add"
     collection_uuid: str
     add_amount: int
+    with_embeddings: bool
+    with_metadatas: bool
+    with_documents: bool
+
+
+@dataclass
+class CollectionUpdateEvent(TelemetryEvent):
+    name: ClassVar[str] = "collection_update"
+    collection_uuid: str
+    update_amount: int
+    with_embeddings: bool
+    with_metadatas: bool
+    with_documents: bool
+
+
+@dataclass
+class CollectionQueryEvent(TelemetryEvent):
+    name: ClassVar[str] = "collection_query"
+    collection_uuid: str
+    with_metadata_filter: bool
+    with_document_filter: bool
+    query_size: int
+    n_neighbors: int
+    including: str
```

Comment:
should we change this to per-field flags to match the others but also to make it faster to query when we do analytics?